l5kit/l5kit/geometry/transform.py

```diff
@@ -92,14 +92,11 @@ def transform_points(points: np.ndarray, transf_matrix: np.ndarray) -> np.ndarra
     ), f"transf_matrix ({transf_matrix.shape}) should be a square matrix."
 
     if points.shape[1] not in [2, 3]:
-        raise AssertionError("Points input should be (N, 2) or (N, 3) shape, received {}".format(points.shape))
+        raise AssertionError(f"Points input should be (N, 2) or (N, 3) shape, received {points.shape}")
 
     assert points.shape[1] == transf_matrix.shape[1] - 1, "points dim should be one less than matrix dim"
 
-    num_dims = len(transf_matrix) - 1
-    transf_matrix = transf_matrix.T
-
-    return points @ transf_matrix[:num_dims, :num_dims] + transf_matrix[-1, :num_dims]
+    return (points @ transf_matrix.T[:-1, :-1]) + transf_matrix[:-1, -1]
```

Comment:
I don't really understand why this should be faster (maybe some memory alignment) but I'm fine with it :)